frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -630,13 +641,19 @@ function VirtualKeyPopup:init()
     }
     if position_container.dimen.x < 0 then
         position_container.dimen.x = 0
+        -- We effectively move the popup, which means the key underneath our finger is no longer parent_key.
```

Comment:
`is no longer parent_key` => `may not be...` (offset may be a fraction of a key width, so we may or may not be the parent key). Still work in that case ?

Reply:
That's still an offset, so, technically, yeah.

Reply:
OK, first commit seems to work for me, thanks :)